src/docker.rs

```diff
@@ -215,6 +207,16 @@ WORKDIR /home/solana
         Ok(())
     }
 
+    fn check_copy_ledger(validator_type: &ValidatorType) -> String {
+        match validator_type {
+            ValidatorType::Bootstrap | ValidatorType::RPC => {
+                "COPY --chown=solana:solana ./config-k8s/bootstrap-validator /home/solana/ledger"
```

Comment:
technically no. it can get the ledger through rpc. i went back and forth on this but i think its better to have all `Standard` validators pull the genesis through RPC. this will minimize the docker image size. but open to your thoughts!

Reply:
Ah gotcha, sorry, I misread this. I was focused on the `chown` bit, and didn't look at the `COPY` bit -- thanks for the explanation!